data/sql/updates/pending_db_world/spawntimeandspeed.sql

```diff
@@ -4,9 +4,32 @@ UPDATE `creature_template` SET `speed_walk` = 1 WHERE (`entry` = 3792);
 -- Akkrilus
 UPDATE `creature` SET `spawntimesecs`=75600 WHERE `guid`=51883 AND `id1`=3773;
 UPDATE `creature_template` SET `speed_walk` = 1 WHERE (`entry` = 3773);
+
+INSERT INTO `creature` (`guid`, `id1`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `VerifiedBuild`) VALUES
+(52022, 3773, 1, 0, 0, 1, 1, 1, 2286.01953125, 44.71516799926757812, 102.4616622924804687, 1.413716673851013183, 75600, 0, 0, 684, 693, 0, 0, 0, 0, 46368);
+
+DELETE FROM `pool_template` WHERE `entry`=1077 AND `description`='Akkrilus Rare Spawn';
+INSERT INTO `pool_template` (`entry`, `max_limit`, `description`) VALUES (1077, 1, 'Akkrilus Rare Spawn');
```

Comment:
There isn't anything standardized for descriptions but most entries in `pool_creature` use 'Name (Entry/Id)'
In my opinion since pools also can take chances into account then 'rare' can be ambiguous 😛